pdfmaker/kassia.py

```diff
@@ -44,10 +45,15 @@ def setDefaults(self):
         self.pageAttrib['line_height'] = 72
         self.pageAttrib['line_width'] = self.pageAttrib['paper_size'][0] - (self.pageAttrib['left_margin'] + self.pageAttrib['right_margin'])
```

Comment:
So I'm thinking that the default attributes will probably end up being paper size dependent. That's a PR for another day, but let me know what you think.